src/test/java/com/google/javascript/cl2dts/nested_enums.js

```diff
@@ -0,0 +1,24 @@
+goog.provide('nested.bar.Baz');
+goog.provide('nested.bar.Baz.NestedEnum');
+goog.provide('nested.bar.SuperBaz');
+goog.provide('nested.bar.SuperBaz.NestedEnum');
+
+/** @constructor */
+nested.bar.Baz = function() {};
+
+/** @enum */
+nested.bar.Baz.NestedEnum = {
+  A: 1
+};
+
+/**
+ * @constructor
+ * @extends {nested.bar.Baz}
+ */
+nested.bar.SuperBaz = function() {};
+
+/** @enum */
+nested.bar.SuperBaz.NestedEnum = nested.bar.Baz.NestedEnum;
```

Comment:
True, I wasn't sure what is the issue at first, so I faithfully reproduced the closure setup I saw. But indeed the real issue is aliased enums, so I will remove the superclassing and rename the test to `aliased_enums`.